script/utils/bookUtils.ts

```diff
@@ -1,21 +1,22 @@
 import https from 'https';
 import fetch from 'node-fetch';
 import { makeUnifiedBookLoader } from '../../src/app/content/utils';
+import { AppServices } from '../../src/app/types';
 import { assertDefined } from '../../src/app/utils';
 import config from '../../src/config';
-import createArchiveLoader from '../../src/gateways/createArchiveLoader';
-import createOSWebLoader from '../../src/gateways/createOSWebLoader';
 
 export async function findBooks({
-  rootUrl,
-  archiveUrl,
+  archiveLoader,
   bookId,
   bookVersion,
+  osWebLoader,
+  rootUrl,
 }: {
-  rootUrl: string,
-  archiveUrl?: string,
+  archiveLoader: AppServices['archiveLoader'],
   bookId?: string,
   bookVersion?: string,
+  osWebLoader: AppServices['osWebLoader']
```

Comment:
loaders could be next to each other and rootUrl may be still first parameter
they don't have to be in alphabetic order and it will create less diff